purchases/src/main/kotlin/com/revenuecat/purchases/google/usecase/GetBillingConfigUseCase.kt

```diff
@@ -35,7 +35,7 @@ internal class GetBillingConfigUseCase(
                     )
                     return@getBillingConfigAsync
                 }
-                processResult(result, config)
+                processResult(result, config, ::onOk)
```

Comment:
oh yes, it's not needed because it's the default of the function

```
    fun processResult(
        billingResult: BillingResult,
        response: T,
        onSuccess: (T) -> Unit = ::onOk,
        onError: (BillingResult) -> Unit = ::forwardError,
    ) {
```

I added that `onSuccess` parameter everywhere, then the default and forgot to remove the parameter here. It's not needed